skipruntime-ts/src/Extern.sk

```diff
@@ -277,6 +277,32 @@ fun mapReduce(
     .toString()
 }
 
+@export("SkipRuntime_union")
+fun union(
+  context: mutable Context,
+  name: String,
+  collections: SKJSON.CJArray,
+): String {
+  mappings = collections match {
+  | SKJSON.CJArray(v) ->
+    v.map(handleID -> {
+      ehandle = EHandle(
+        JSONID::keyType,
+        JSONFile::type,
+        DirName::create(SKJSON.asString(handleID)),
+      );
+      id_mapper = (_ctx, writer, key, vals) ~> {
+        //identity mapper function: write values straight through
+        writer.setArray(key, vals.collect(Array))
+      };
```

Comment:
Let's share our identity mappers (see #321)

Reply:
Hey no fair, I wrote mine first ;)

Sounds good, I'll rebase and do that once yours lands in trunk